website/docs/reference/warehouse-setups/postgres-setup.md

```diff
@@ -95,6 +95,25 @@ The `role` config controls the Postgres role that dbt assumes when opening new c
 
 The `sslmode` config controls how dbt connectes to Postgres databases using SSL. See [the Postgres docs](https://www.postgresql.org/docs/9.1/libpq-ssl.html) on `sslmode` for usage information. When unset, dbt will connect to databases using the Postgres default, `prefer`, as the `sslmode`.
 
+
+#### sslcert
+
+<Changelog> Added in v0.21.0 </Changelog>
+
+The `sslcert` config controls the location of the certificate file used to connect to Postgres when using client SSL connections. To use a certificate file that is not in the default location, set that file path using this value. Without this config set, dbt uses the Postgres default locations. See the [Postgres SSL Support](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-CLIENTCERT) page for the default paths.
```

Comment:
sugg using the exact title of the section and calling it out that it's in the postgresql docs (for clarity in case people might assume it's in the dbt docs)

```suggestion
The `sslcert` config controls the location of the certificate file used to connect to Postgres when using client SSL connections. To use a certificate file that is not in the default location, set that file path using this value. Without this config set, dbt uses the Postgres default locations. See [Client Certificates](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-CLIENTCERT) in the PostgreSQL SSL docs for the default paths.
```

Reply:
Great suggestion!